pkg/specs/podspec_diff_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package specs
+
+import (
```

Comment:
bit of a nit here, but we already have existing PodSpec tests, in `pods_test.go`. We might consider moving this there, or possibly even better, bringing over the PodSpec tests to this file.

Reply:
I agree it would be better. Let's do it in another PR.